### ISSUE TITLE
ExtractorError: lynda returned error: Video not found

### ISSUE THREAD

**Issue by boazrymland** [rg3/youtube-dl#5093]
Hi,

More details:
**Course URL**: http://www.lynda.com/Bootstrap-tutorials/Bootstrap-3-Adding-Interactivity-Your-Site/161098-2.html  
**Version used**: 2015.02.28

**Command line**: _(sans url, username + password)_ 
> -w -o "%(autonumber)s__%(title)s.%(ext)s" --autonumber-size 3 --verbose

**Verbose output**:

```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-w', '-o', '%(autonumber)s__%(title)s.%(ext)s', '--autonumber-size', '3', '--verbose', '--username=XXX', '--password=XXX', 'http://www.lynda.com/Android-tutorials/Android-SDK-Essential-Training/143102-2.html']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.02.28
[debug] Python version 2.7.6 - Linux-3.13.0-46-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: avconv 9.16-6, avprobe 9.16-6, rtmpdump 2.4
[debug] Proxy map: {}
[lynda:course] 143102: Downloading course JSON
[download] Downloading playlist: Android SDK Essential Training
[lynda:course] playlist Android SDK Essential Training: Collected 56 video ids (downloading 56 of them)
[download] Downloading video 1 of 56
[lynda] Logging in as XXX
[lynda] Confirming log in and log out from another device
[lynda] 158773: Downloading video JSON
[lynda] 158773: Checking 540 video format URL
[lynda] 158773: Checking 720 video format URL
[lynda] 158773: Checking 360 video format URL
[debug] Invoking downloader on u'http://files3.lynda.com/secure/courses/143102/VBR_MP4h264_main_HD720/143102_00_01_WX30_welcome.mp4?4d76751a3bd5908753e46e419fc885a087f1591cebfb0a875aba30e02e6a04c766c83f0bbd930cce5b163ffe6a45970c9e538fa4c008b474404c008b338bfb195e6853ecc2bd5b4366a6d802f2ab531ec7c4b95b6aa0378f9f1bea30774cabfaf51fd4513e2530de098c94cd89f5dec58d552922ab02c681378b216721268154b9&c3.ri=3773259075705769528'
[download] 001__Welcome.mp4 has already been downloaded
[download] 100% of 8.04MiB
[download] Downloading video 2 of 56
[lynda] 158774: Downloading video JSON
[lynda] 158774: Checking 540 video format URL
[lynda] 158774: Checking 720 video format URL
[lynda] 158774: Checking 360 video format URL
[debug] Invoking downloader on u'http://files3.lynda.com/secure/courses/143102/VBR_MP4h264_main_HD720/143102_00_02_XR15_prereqs.mp4?4d76751a3bd5908753e46e419ec885a0857a0a844cb833889b58807818c54e32f531a4779f3efbc8eebfb95fe85aa1394723b6b42bc56735971f8004f9da074428ccc161486e598c74855bccae5554637cd955506d279edfdcc161d3e9fcb3288f3e42308e084687f6babc17031e0a84291983830bf5977dfb291ea634522a96&c3.ri=3773259075705769547'
[download] Destination: 002__What you need to know before starting this course.mp4
[download] 100% of 4.95MiB in 00:01
[download] Downloading video 3 of 56
[lynda] 158775: Downloading video JSON
[lynda] 158775: Checking 540 video format URL
[lynda] 158775: Checking 720 video format URL
[lynda] 158775: Checking 360 video format URL
[debug] Invoking downloader on u'http://files3.lynda.com/secure/courses/143102/VBR_MP4h264_main_HD720/143102_00_03_XR15_exfiles.mp4?4d76751a3bd5908753e46e419fc885a087f1591cebfb0a875aba30e02e6a04c766cc3f08b3940dc3d4463b72d302ca5fea9225daaac60d94ec137b14465cb8e64971b3f1a3c383ffd9166a5c260d5f747d75768507f002f2a76e8e979678b79c3038dc8f9d9f239383e743ee948ced05b26fecc17e4661d742c7708c1d1ed1f7db&c3.ri=3773259075705769574'
[download] Destination: 003__Using the exercise files.mp4
[download] 100% of 4.24MiB in 00:00
[download] Downloading video 4 of 56
[lynda] 158777: Downloading video JSON
ERROR: lynda returned error: Video not found
Traceback (most recent call last):
  File "/tmp/youtube-dl_local/youtube-dl/youtube_dl/YoutubeDL.py", line 643, in extract_info
    ie_result = ie.extract(url)
  File "/tmp/youtube-dl_local/youtube-dl/youtube_dl/extractor/common.py", line 275, in extract
    return self._real_extract(url)
  File "/tmp/youtube-dl_local/youtube-dl/youtube_dl/extractor/lynda.py", line 56, in _real_extract
    raise ExtractorError('lynda returned error: %s' % video_json['Message'], expected=True)
ExtractorError: lynda returned error: Video not found


```

**Comment by boazrymland** [comment]
Please put this on hold - it might have been a simple subscription expired issue. Still checking it (if relevant, I'll renew it and then check again).

**Comment by dstftw** [comment]
This is due to insufficient access rights of your account (yes, probably due to expired subscription). In the next version all videos that your current account has no rights to access will be skipped. Thanks for the report.
